homeassistant/components/knx.py

```diff
@@ -45,6 +51,12 @@
     vol.Required(CONF_KNX_LOCAL_IP): cv.string,
 })
 
+EXPOSE_SCHEMA = vol.Schema({
+    vol.Required(CONF_KNX_EXPOSE_TYPE): cv.string,
```

Comment:
Do you know these possible types already? If so validate that the type is one of them.

Reply:
well, it is a growing list of 30 different types (KNX itself has about 100 different types). And same list as it is used within `sensor/knx.py`. I tend to say maintaining this list on 3 different places is too much overhead and is error-prone.

Or is there any possibility to us an external lib within validation?

Reply:
Requirements are not installed at the time of validation, last time I looked. So 3rd party library usage during validation is not possible.

Ok. Just curious what's the third place to maintain a list? I understand python knx library and homeassistant are two.

Reply:
https://github.com/XKNX/xknx/blob/master/xknx/devices/remote_value_sensor.py#L22